include/amici/sundials_matrix_wrapper.h

```diff
@@ -117,10 +129,16 @@ class SUNMatrixWrapper {
     sunindextype columns() const;
 
     /**
-     * @brief Get the number of non-zero elements (sparse matrices only)
+     * @brief Get the number of specified non-zero elements (sparse matrices only)
      * @return number
      */
     sunindextype nonzeros() const;
+    
+    /**
+     * @brief Get the number of allocated non-zero elements (sparse matrices only)
+     * @return number
+     */
+    sunindextype nonzero_space() const;
```

Comment:
```suggestion
    sunindextype capacity() const;
```

To be coherent with std::vector?